content/en-us/tutorials/fundamentals/coding-4/repeating-code-with-while-loops.md

```diff
@@ -25,14 +25,14 @@ Use any Roblox project for this exercise. When finished, this part can be used a
 
 ## Using Variables to Store a Part
 
-Not only can variables hold strings, they can also hold a reference, or directions, to an object. Once a script knows the direction to a part, it can affect changes to it.
+Not only can variables hold strings, they can also hold a reference, or directions, to an object. Once a script knows the direction to a part, it can effect changes to it.
```

Comment:
I believe this one is `affect`. The script affects changes to the part.

- https://www.grammarly.com/blog/affect-vs-effect/
- https://www.merriam-webster.com/grammar/affect-vs-effect-usage-difference

```suggestion
Not only can variables hold strings, they can also hold a reference, or directions, to an object. Once a script knows the direction to a part, it can affect changes to it.
```

But this brings up a good point. Maybe we can make this sentence even clearer. What do you think about this:

```suggestion
Variables can also hold references to objects besides strings. For example, when a script knows the reference to a part, the script can change it.
```